student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student
```

Comment:
- [ ] The requirement says `Student` Inherits from `Person`. You need a code like this: `class Student < Person`
- [ ] Please [open this link to see examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.)

Reply:
Thanks a lot, I totally forgot to inherit the super class!

Reply:
well resolved ✔️